rle_array/autoconversion.py

```diff
@@ -86,3 +101,30 @@ def auto_convert_to_rle(df: pd.DataFrame, threshold: float = 1.0) -> pd.DataFram
         data[col] = array_target
 
     return pd.DataFrame(data, index=index)
+
+
+def decompress(df: pd.DataFrame, threshold: Optional[float] = None) -> pd.DataFrame:
+    """
+    Decompress all RLE columns in the provided DataFrame.
+
+    Parameters
+    ----------
+    df
+        Input DataFrame. This input data MIGHT not be copied!
+    """
+    index = df.index
+
+    data = {}
+    for col in df.columns:
+        series = df[col]
+        array = series.array
+        dtype = series.dtype
+
+        if _is_rle_dtype(dtype):
```

Comment:
decompression can never fail?

Reply:
it cannot. RLE arrays are backed by numpy arrays which are all supported by pandas.